composer.json

```
@@ -5,7 +5,7 @@
     "license": "MIT",
     "homepage": "https://cardinity.com",
     "require": {
-        "php"                       : ">=7.2.5",
+        "php"                       : ">=5.6",
         "guzzlehttp/guzzle"         : "^6.2.1",
         "guzzlehttp/oauth-subscriber": "0.3.*",
         "symfony/validator"         : "^5.0.0"
```

```suggestion
        "symfony/validator"         : "^3.0.0 || ^4.0.0 || ^5.0.0"
```

Only Symfony Validator supports PHP 5.6.

---

@garas thank you for participating in development process. We are closing this PR for now.